docs/api-reference/global-functions/Integer.md

```diff
@@ -1,17 +1,23 @@
 ---
-description: Integer Integer(Integer value)
+uid: crmscript_global_integer
+title: Integer
+description: CRMScript Integer constructor
 intellisense: Void.Integer
 langref: 1
-keywords: Integer(Integer)
+keywords: Integer(Integer), Integer(String value)
 so.topic: reference
 ---
 
```

Comment:
Same comment as Float and Date..
Consider adding link to or examples from [Integer data type docs](https://github.com/SuperOfficeDocs/crmscript/blob/main/docs/datatypes/integer-type.md)